indexer/tasks/src/multiera/dex/minswap_v1.rs

```diff
@@ -24,18 +25,23 @@ impl Dex for MinSwapV1 {
     fn queue_mean_price(
         &self,
         queued_prices: &mut Vec<QueuedMeanPrice>,
-        tx: &MultiEraTx,
+        tx: &cml_multi_era::MultiEraTransactionBody,
+        tx_witness: &cml_chain::transaction::TransactionWitnessSet,
         tx_id: i64,
     ) -> Result<(), String> {
         // Note: there should be at most one pool output
         if let Some((output, datum)) = filter_outputs_and_datums_by_hash(
             &tx.outputs(),
             &[POOL_SCRIPT_HASH1, POOL_SCRIPT_HASH2],
-            &tx.plutus_data(),
+            &tx_witness.plutus_datums.clone().unwrap_or_default(),
         )
         .first()
         {
-            let datum = datum.to_json();
+            let pallas_datum = pallas::ledger::primitives::alonzo::PlutusData::decode_fragment(
```

Comment:
replace this with the JSON conversion in CML. You may need https://github.com/dcSpark/cardano-multiplatform-lib/pull/300

Reply:
how do I import that? I can't see a feature / method similar to `to_json`

Reply:
fixed